### PR TITLE
created Calculator component

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I implemented the following:

- Deleted all the boilerplate from CRA (text, images, styles).

- Created a directory called components.

- Inside components, created a new Calculator.js file and added a React component that matches the design provided.

- Developed the UI of the calculator.

- Used class-based components.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>My Calculator</h1>
+      <Calculator />
     </div>
```
Comment (AshakaE) [inline src/App.js]: The `App.js` should be class based.

- [x] Kindly update the `App.js` component

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @themonster2015  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (themonster2015) [reply src/App.js]: I just fixed it

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @themonster2015  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**